base_classes/NXwaveplate.nxdl.xml

```diff
@@ -74,48 +75,53 @@
     </field>
     <field name="wavelengths" type="NX_NUMBER" recommended="true">
         <doc>
-             Discrete wavelengths for which the waveplate is designed. If the
-             waveplate operates over an entire range of wavelengths, enter the minimum
-             and maximum values of the wavelength range (in this case
-             N_wavelengths = 2).
+            Discrete wavelengths for which the waveplate is designed. If the
+            waveplate operates over an entire range of wavelengths, enter the minimum
+            and maximum values of the wavelength range (in this case
+            N_wavelengths = 2).
```

Comment:
Add a reminder that type="achromatic" too.

Reply:
I checked if the implementation in the classes and appdefs that was voted upon resolved this comment - it does.